scipy/stats/_continuous_distns.py

```diff
@@ -2556,7 +2556,17 @@ class gamma_gen(rv_continuous):
 
     When :math:`a` is an integer, `gamma` reduces to the Erlang
     distribution, and when :math:`a=1` to the exponential distribution.
+    
+    Gamma distributions are sometimes parameterized with two variables,
+    with a probability density function of:
 
+    .. math::
+
+        f(x, \alpha, \beta) = \frac{\beta^\alpha x^{\alpha - 1} e^{-\beta x }}{\Gamma(\alpha)}
+
+    Note that this parameterization is equivalent to the above, with 
+    :math:`scale = 1 / beta`.
```

Comment:
Let's do one of these

`scale = 1 / beta` (so scale formatting matches below)
or
:math:\`scale = 1 / \beta\` (so beta matches formula above)